src/pretix/presale/forms/checkout.py

```diff
@@ -152,6 +152,9 @@ def _label(self, event, item_or_variation, avail, override_price=None):
             n += ' – {}'.format(_('SOLD OUT'))
         elif avail[0] < 100:
             n += ' – {}'.format(_('Currently unavailable'))
+        else:
+            if avail[1] is not None:
```

Comment:
Replace by:
```
if avail[1] is not None and event.settings.show_quota_left:
```
and this is already fixing #798 completely :)

Reply:
Sorry, I totally forgot about this setting.